ui/lib/utils/tableColumnFactory.tsx

```diff
@@ -0,0 +1,256 @@
+      tooltipPrefix: Object.keys(max_)[0],
+    }
+    tooltioPrefixLens.push(max.tooltipPrefix.length)
+    let min
+    if (min_) {
+      min = {
+        fieldName: Object.values(min_)[0],
+        tooltipPrefix: Object.keys(min_)[0],
+      }
+      tooltioPrefixLens.push(min.tooltipPrefix.length)
+    } else {
+      min = undefined
+    }
+    const maxTooltipPrefixLen = _max(tooltioPrefixLens) || 0
+
+    const capacity = rows ? _max(rows.map((v) => v[max.fieldName])) ?? 0 : 0
+    let refFields = [avg.fieldName, max.fieldName]
+    if (min) {
+      refFields.push(min.fieldName)
+    }
+    return {
+      name: this.columnName(displayTransKey || avg.fieldName),
+      key: avg.fieldName,
+      fieldName: avg.fieldName,
+      refFields,
+      minWidth: 140,
+      maxWidth: 200,
+      columnActionsMode: ColumnActionsMode.clickable,
+      onRender: (rec) => {
+        const avgVal = rec[avg.fieldName]
+        const maxVal = rec[max.fieldName]
+        const minVal = min ? rec[min.fieldName] : undefined
+        const tooltips = [avg, min, max]
+          .filter((el) => el !== undefined)
+          .map((bar) => {
+            const prefix = capitalize(bar!.tooltipPrefix + ':').padEnd(
+              maxTooltipPrefixLen + 2
+            )
+            const fmtVal = formatVal(rec[bar!.fieldName], unit)
+            return `${prefix}${fmtVal}`
+          })
+          .join('\n')
+        return (
+          <Tooltip title={<Pre>{tooltips.trim()}</Pre>}>
+            <Bar
+              textWidth={70}
+              value={avgVal}
+              max={maxVal}
+              min={minVal}
+              capacity={capacity}
+            >
+              {formatVal(avgVal, unit)}
+            </Bar>
+          </Tooltip>
+        )
+      },
+    }
+  }
+
+  timestamp(fieldName: string): IExtendColumn {
+    return {
+      name: this.columnName(fieldName),
+      key: fieldName,
+      fieldName: fieldName,
+      minWidth: 100,
+      maxWidth: 150,
+      columnActionsMode: ColumnActionsMode.clickable,
+      onRender: (rec) => (
+        <TextWrap>
+          <DateTime.Calendar unixTimestampMs={rec[fieldName] * 1000} />
+        </TextWrap>
+      ),
+    }
+  }
+
+  sqlText(fieldName: string, showFullSQL?: boolean): IExtendColumn {
+    return {
+      name: this.columnName(fieldName),
+      key: fieldName,
+      fieldName: fieldName,
+      minWidth: 100,
+      maxWidth: 500,
+      isMultiline: showFullSQL,
+      onRender: (rec) =>
+        showFullSQL ? (
+          <TextWrap multiline>
+            <HighlightSQL sql={rec[fieldName]} />
+          </TextWrap>
+        ) : (
+          <Tooltip
+            title={<HighlightSQL sql={rec[fieldName]} theme="dark" />}
+            placement="right"
+          >
+            <TextWrap>
+              <HighlightSQL sql={rec[fieldName]} compact />
+            </TextWrap>
+          </Tooltip>
+        ),
+    }
+  }
+
+  plan(fieldName: string): IExtendColumn {
```

Comment:
We can remove this column template (and not display it in the list as well). This use case is too rare.